common/dal2/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "common-dal2"
+version = "0.1.0"
+authors = ["Databend Authors <opensource@datafuselabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[lib]
+doctest = false
+test = false
+
+[dependencies]
+common-base = { path = "../base" }
+common-datablocks = { path = "../datablocks" }
+common-exception = { path = "../exception" }
+common-infallible = { path = "../infallible" }
+
+async-compat = "0.2.1"
+async-trait = "0.1.52"
+azure_core_mirror = "0.1.0"
+azure_storage_mirror = { version = "0.1.0", features = ["blob"] }
+bytes = "1.1.0"
+futures = "0.3.19"
+metrics = "0.17.1"
+reqwest = "0.11.7"
+rusoto_core = "0.47.0"
+rusoto_s3 = "0.47.0"
+rusoto_sts = "0.47.0"
+rusoto_credential = "0.47.0"
+tokio = "1.15.0"
+
+[dev-dependencies]
```

Comment:
It does not appear to use these dependencies.

Reply:
Yep, I will update it.